modules/rux9p/src/netdev.rs

```diff
@@ -7,13 +7,14 @@
  *   See the Mulan PSL v2 for more details.
  */
 use axnet::TcpSocket;
+use axsync::Mutex;
 use core::net::{IpAddr, Ipv4Addr, SocketAddr};
 use driver_9p::_9pDriverOps;
 use driver_common::{BaseDriverOps, DeviceType};
 use log::*;
 
 pub struct Net9pDev {
-    socket: TcpSocket,
+    socket: Mutex<TcpSocket>,
```

Comment:
Why should this be wrapped with `Mutex`.

Reply:
Because lwip TcpSocket has member converted from c , so it  cannot be shared between threads safely